### PR TITLE
Add additional information to topic pages

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
**Unsigned commit in this branch - use squash merge when merging**

**This will need an a11y swarm**

Resolves WSTEAMA-72

**Overall change:**
Adds image badge and description to Topic pages with this data.

**Code changes:**

- Wraps the h1, img and paragraph in a wrapper
- Moves padding from h1 to wrapper
- Adds image component with badge
- Adds description

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/TopicPage/TopicImage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  max-width: 100%;
+  max-height: 100%;
+  object-fit: cover;
+  border-radius: 50%;
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+  }
+`;
+
+const ImageWrapper = styled.div`
+  width: 64px;
```
Comment (greenc05) [inline src/app/pages/TopicPage/TopicImage/index.jsx]: Please could we use relative units?

File: src/app/pages/TopicPage/TopicImage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  height: 64px;
```
Comment (greenc05) [inline src/app/pages/TopicPage/TopicImage/index.jsx]: Please could we use relative units?

File: src/app/pages/TopicPage/TopicPage.jsx [hunk 3]
```diff
@@ -85,8 +94,18 @@ const TopicPage = ({ pageData }) => {
           headline={title}
           entities={promoEntities}
         />
-
-        <TopicTitle>{title}</TopicTitle>
+        <TitleWrapper>
+          {images && (
+            <TopicImage image={images[0].url} altText={images[0].altText} />
```
Comment (greenc05) [inline src/app/pages/TopicPage/TopicPage.jsx]: According to the screen reader UX, I think the alt text should always be empty?

File: data/pidgin/topics/c95y35941vrt.json [hunk 4]
```diff
@@ -1,6 +1,15 @@
 {
   "data": {
     "title": "Donald Trump",
+    "description": "Donald John Trump is an American politician, media personality, and businessman who served as the 45th president of the United States from 2017 to 2021.",
+    "images": [{
+      "url": "https://ichef.bbci.co.uk/news/480/cpsdevpb/484e/test/7aa8e800-f95c-11ec-8adf-7bf4017ee9f9.jpg",
+      "urlTemplate": "https://ichef.bbci.co.uk/news/{width}/cpsdevpb/484e/test/7aa8e800-f95c-11ec-8adf-7bf4017ee9f9.jpg",
+      "originalSize": { "width": 450, "height": 253 },
+      "altText": "Penguins",
```
Comment (greenc05) [inline data/pidgin/topics/c95y35941vrt.json:9]: Not sure you need this to be populated?

File: src/app/pages/TopicPage/TopicDescription/index.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  GEL_GROUP_3_SCREEN_WIDTH_MIN,
+  GEL_GROUP_4_SCREEN_WIDTH_MIN,
+} from '#legacy/gel-foundations/src/breakpoints';
+import {
+  GEL_SPACING,
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '#legacy/gel-foundations/src/spacings';
+
+const Paragraph = styled.p`
```
Comment (greenc05) [inline src/app/pages/TopicPage/TopicDescription/index.jsx:17]: Please could we pass the text colour to this directly?

**Review by greenc05 — COMMENTED** [review]
Wondering if we can load a smaller image to save bytes? When CSS is off I believe it's larger than it needs to be?

**Review by RetiredBlob — APPROVED** [review]

**Review by pvaliani — APPROVED** [review]
Looks good! nice one :)

**Review by RetiredBlob — APPROVED** [review]

**Pull request merged**